recipes/torchmd-net/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "torchmd-net" %}
+{% set version = "0.8.0" %}
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchmd/torchmd-net/archive/{{ version }}.tar.gz
+  sha256: 0f1ff18cd9c860f57f3ad588f42c20aa7f8de97bbf0fa84ca78816d6b77fe08e
+
+build:
+  entry_points:
+    - torchmd-train = torchmdnet.scripts.train:main
+  number: 0
+# Mainly due to NNPOps only having a linux package
+  skip: true  # [not linux]
+  rpaths:
```

Comment:
Is this really needed? If so: Can you post the error? Mostly it works without the rpaths spec.

Reply:
Thank you very much for your review @xhochy !
The error is this here:
https://github.com/conda-forge/staged-recipes/pull/24021#issuecomment-1727782866
I will try again and see if the error has been solved since then.

Reply:
I tried to remove the lines and the old error is reproduced, the package compiles fine, but then this is printed:
```shell


Packaging torchmd-net
Packaging torchmd-net-0.8.0-py39h42bb6ba_0
compiling .pyc files...
number of files: 87
Warning: rpath /home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold (removing it)
   INFO: sysroot: '/home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu', '/home/conda/staged-recipes/build_artifacts/torchmd-net_1698057723522/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/WET']'
  ERROR (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): $RPATH/libc10.so not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
  ERROR (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): $RPATH/libtorch_cpu.so not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
   INFO (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-13.2.0-h7e041cc_2
   INFO (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-13.2.0-h807b86a_2
   INFO (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_16
WARNING (torchmd-net): interpreter (Python) package conda-forge::python-3.9.18-h0755675_0_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (torchmd-net): run-exports library package conda-forge::pytorch-2.0.0-cpu_mkl_py39h1d6e76c_103 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 228, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 118, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 174, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 253, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3804, in build_tree
    packages_from_this = build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2880, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2006, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1817, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1721, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1605, in check_overlinking
    return check_overlinking_impl(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1576, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
['  ERROR (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): $RPATH/libc10.so not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (torchmd-net,lib/python3.9/site-packages/torchmdnet/extensions/torchmdnet_extensions.so): $RPATH/libtorch_cpu.so not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?']
##[error]Bash exited with code '1'.
Finishing: Run docker build
```

Reply:
`pytorch` has its libraries not in the standard place.